### DISCUSSION TITLE
How to set ssrc when create offer or create answer

### DISCUSSION THREAD

**Discussion by ChenMoGe2** [pion/webrtc#2049] [Q&A]
### Your environment.
  - Version: v3.1.11
  - Browser: Android 9

### What did you do?
I make a sfu-server using pion webrtc which create peer connection for each client.And the client judges whose is speech using ssrc in received rtp packet.So how I set ssrc for the track

**Comment by ChenMoGe2** [comment]
```
// writeRTP is like WriteRTP, except that it may modify the packet p
func (s *TrackLocalStaticRTP) writeRTP(p *rtp.Packet) error {
	s.mu.RLock()
	defer s.mu.RUnlock()

	writeErrs := []error{}

	for _, b := range s.bindings {
		//p.Header.SSRC = uint32(b.ssrc)
		p.Header.PayloadType = uint8(b.payloadType)
		if _, err := b.writeStream.WriteRTP(&p.Header, p.Payload); err != nil {
			writeErrs = append(writeErrs, err)
		}
	}

	return util.FlattenErrs(writeErrs)
}
```

I delete the code 
```
p.Header.SSRC = uint32(b.ssrc)
```
so that the ssrc not change and the client can receive correct ssrc.So how can I forward the rtp packet not change to use binding ssrc but use the original ssrc in pion webrtc lib?

**Comment by Sean-Der** [comment]
Hey @ChenMoGe2 

The PeerConnection needs to generate a SSRC for each track. When we do this we make sure you don't have any SSRC collisions.

 When you create the tracks could you use the ID/Label instead to communicate the original speaker instead?

**Reply by ChenMoGe2** [reply]
@Sean-Der 
Hi.Thank you for your reply.I know use ID/Label can distinguish the track at server side and client side,but my signal server doesn't exchange traditional SDP information.In order to reduce the content length of SDP,it just need exchange SSRC and the signal server can control the SSRC unique.The main reason I use SSRC distinguish different track is the RTP packets just contain the SSRC information but not the ID/Label.
